### PR TITLE
Updates to env, base/functions and testenv metadata

### PR DESCRIPTION
**Pull request by michaeldisaro**
* Added zendesk support services env variables
* updated base/functions to point functions-node-14
* updated spid metadata to work with new certs

### CODE REVIEW

**Review by gquadrati — CHANGES_REQUESTED** [review]
Just a little comment.

File: base/functions/Dockerfile [hunk 1]
```diff
@@ -1,2 +1,2 @@
-FROM functions
+FROM functions-node-14
```
Comment (gquadrati) [inline base/functions/Dockerfile:1]: This is gonna update node in all functions.

The idea was to gradually introduce node 14 in each function by changing its [Dockerfile](https://github.com/pagopa/io-mock/blob/df6deafe4e3bb05d828b7762b2628080b635d9e8/io-functions-eucovidcerts/Dockerfile#L1) and the [docker-compose.yml](https://github.com/pagopa/io-mock/blob/df6deafe4e3bb05d828b7762b2628080b635d9e8/docker-compose.yml#L263)

Comment (michaeldisaro) [reply base/functions/Dockerfile:1]: This is breaking the docker compose override. As far as I know functions should all be migrated to node 14. Can you spot which remained on node 10?

Comment (gquadrati) [reply base/functions/Dockerfile:1]: >  As far as I know functions should all be migrated to node 14

True, the idea was to move all of them to function-node-14 and then remove function.

**Pull request merged**